### ISSUE TITLE
No webhook handlers configured

### ISSUE THREAD

**Issue by panayotoff** [Bottr-js/Bottr#77]
Hi;

I tried the example app, but have some problems making it work. From the deploying page:

> In the Webhooks section click “Setup Webhooks”, for the callback URL you need to enter “/webhook” and for the verify token enter in your validation secret you added to your bot.

I guess it should be absolutely url ( otherwise facebook will throw an error );

However when I try this I get this response from my url:
```
{
"error": "No webhook handlers configured"
}
```

The interesting thing is the bot is working when receiving first message from the first user, then it stop working.

**Comment by jcampbell05** [comment]
@panayotov I'm wondering if there is somehting about FB's API that changed. I'm working on making the client more robust. 

Are you able to get a log of what Facebook sends to your server ?

**Comment by panayotoff** [comment]
@jcampbell05 No, just this. I gave up eventually, but I'll give it a try as soon as possible.
